src/python/WMCore/MicroService/MSPileup/MSPileupData.py

```diff
@@ -45,6 +45,21 @@ def stripKeys(docs, skeys=None):
     return docs
 
 
+def getNewTimestamp(doc):
```

Comment:
this function can be simplified since you do not need to create new dictionary and return it if you pass a dictionary object. Instead, you may update the provided dictionary in place.

Reply:
I can change this function to actually update the dictionary in place, however you are always the first one to jump in and ask for specific data return + update, instead update in-place. Please let me know your preference.